pyrealm/splash/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""The splash submodule implements the SPLASH model :cite:p:`davis:2017a` for
+calculating robust indices of radiation, evapotranspiration and plant-available
+moisture.
+
+It is based on the [SPLASH v1.0 implementation](https://doi.org/10.5281/zenodo.376293)
+published with the original description but has been extensively refactored and
+restructured. The main changes are:
+
+* The implementation now provides a programmatic API to the main model components,
+  allowing the functionality to be imported and used within Python workflows. The
+  original code was implemented as command line scripts that required editing for
+  particular use cases.
+
+* The code has been updated to accept numpy arrays and to calculate estimates
+  simulataneously across arrays. The original code predominantly expected scalar inputs
+  and required users to script iteratation over sites.
+
+* The majority of solar and evaporative fluxes do not vary are invariant given the
```

Comment:
```suggestion
* The majority of solar and evaporative fluxes do not vary given the
```

Reply:
Which evaporative fluxes are you referring to? I agree that the radiation can be done in one go at the start, but evaporative fluxes often depend on the soil moisture of the previous time step. I'll probably see later on what variables are affected, but might be worth stating clearly here :-)